### DISCUSSION TITLE
pass build features in recipe?

### DISCUSSION THREAD

**Discussion by jwinarske** [meta-rust/meta-rust#308] [Q&A] [answered]
How does one pass a build feature in a recipe?

**Comment by janderholm** [comment, marked as answer]
Adding this to your recipe should probably do it. Look in libstd-rs.inc for an example. 

CARGO_BUILD_FLAGS += "--features 'a b c'" 

That said, you raise a good point. We should probably change this to use EXTRA_OECARGO and PACKAGECONFIG like most other package instead.

**Reply by jwinarske** [reply]
is the default build release?

**Reply by janderholm** [reply]
Yes, although you can change this by setting DEBUG_BUILD = "1" (this actually is similar to all other recipes!)

**Reply by jwinarske** [reply]
cool thanks!
